pkg/email/templates/system_intake_lcid_expiration_alert.gohtml

```diff
@@ -0,0 +1,46 @@
+{{template "easi_header.gohtml"}}
+
+<p>The Life Cycle ID that was issued for {{.ProjectName}}{{if .LCIDIssuedAt}} on {{.LCIDIssuedAt}}{{end}} is set to expire on {{.ExpiresAt}}. If your Lifecycle ID expires, your project will be operating under an expired Lifecycle ID and will be added to the Capital Planning Investment Control (CPIC) risk register.</p>
+<p>To avoid this, please email the Governance Team at <a href="mailto:{{.GRTEmail}}">{{.GRTEmail}}</a> within one week to update them with the current status of your project.</p>
+<p>
+  For New IT development projects, please include (if applicable):
+  <ul>
+    <li>if the project is in production and if so, the date it was released into production</li>
+    <li>if development of the project is still underway and if so, the target production release date</li>
+    <li>the date the final contract option year expires</li>
+    <li>if the development effort has encountered difficulties and would like technical assistance (please also include the target production date)</li>
+    <li>if the project has been cancelled</li>
+    <li>if the project is on hold</li>
+  </ul>
+</p>
+<p>
+  For O&M projects or services contracts, please include (if applicable):
+  <ul>
+    <li>if the current contract is not being extended, include the end date of the period of performance</li>
+    <li>if a new contract or re-compete is being planned, include the target date for release of solicitation and the target award date</li>
+    <li>if an extension of the current contract is planned, include the new contract expiration date</li>
+    <li>describe any new development or planned changes to service requirements, if any</li>
+    <li>if you anticipate a cost increase, please indicate how much of an increase you anticipate over what you are currently spending</li>
+    <li>if contract support is no longer needed</li>
+  </ul>
+</p>
+<p>
+  View this request in EASi:
+  <ul>
+    <li>The person who initially submitted this request, {{.RequesterName}}, may <a href="{{.RequesterLink}}" style="font-weight: bold">click here</a> to view the request task list and <a href="{{.RequesterLink}}/request-decision" style="font-weight: bold">here</a> to view the decision and LCID information</li>
```

Comment:
[FIGMA](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=2202-12656&mode=design&t=1W2JkzMlxIJZZAaQ-0)
doesn't include a link to decision and LCID information.